spec/system/placements/support/schools/placements/support_user_views_a_placement_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Support / Schools / Placement / Support User views a placement",
+               type: :system, service: :placements do
+  let(:school) { create(:placements_school, name: "School 1", phase: "Nursery") }
+  let(:placement) do
+    create(:placement, school:, mentors: [mentor], subjects: [subject], status: "published", start_date: nil,
+                       end_date: nil)
+  end
+  let(:mentor) { create(:placements_mentor) }
+  let(:subject) { create(:subject, name: "Maths", subject_area: :primary) }
```

Comment:
Could we add a scenario for when multiple subjects are associated with a placement?